qa/integration-tests/src/test/java/io/camunda/it/exporter/IncidentExporterIT.java

```diff
@@ -19,11 +19,15 @@
 import java.util.function.Consumer;
 import org.awaitility.Awaitility;
 import org.junit.jupiter.api.TestTemplate;
-import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
-@ExtendWith(BrokerWithCamundaExporterITInvocationProvider.class)
 public class IncidentExporterIT {
 
+  // RDBMS doesn't support Incidents up to now
+  @RegisterExtension
+  static final BrokerITInvocationProvider PROVIDER =
+      new BrokerITInvocationProvider().withoutRdbmsExporter();
```

Comment:
❓ Does it maybe make more sense only to enable it where we want to run it? I guess it would also be fewer changes? 🤔 Not sure about the compatibility yet.

Reply:
My thoughts with this was that it should be default that we also test against the rdbms exporter. So it should be a manual step to deactivate the exporter. But I'm also fine with doing it the other way around, what do you mean?